_issues/trade.md

```diff
@@ -0,0 +1,30 @@
+---  
+layout: post  
+title: "But isn't she for free trade?"  
+categories: policy
+---  
+
+Q: Doesn't Hillary support trade agreements that take jobs away from American workers?
+  
+A: Not at all. Despite what Donald Trump would have you believe, the world is a complicated place. Trade deals are complex, because they are supposed to benefit all of the nations involved. Hillary believes that if a trade deal does not benefit American workers, it must be opposed.  
+
+Consider the facts:
+
+* Fact: Hillary is neither pro-trade nor anti-trade. Instead, she evaluates each trade deal on its merits and determines the overall benefit to American workers. As [Gene Sperling, director of the National Economic Council in the Clinton and Obama administrations](http://www.politifact.com/truth-o-meter/statements/2016/apr/10/bernie-s/clinton-voted-virtually-every-trade-agreement-kill/) put it:
+
+> Some people are generally pro-trade or anti-trade. She’s case-by-case on trade.
+
+* Fact: As U.S. Senator from New York, Hillary voted for roughly half of trade deals that came before her. However, she voted against the [Dominican Republic-Central America Free Trade Agreement(DR-CAFTA)](https://scout.sunlightfoundation.com/item/speech/CREC-2005-06-30-pt2-PgS7697.chunk96/sen-hillary-clinton-senate), saying:
+
+> While this agreement provides some benefit for New York, I regretfully conclude the harm outweighs the good. I must therefore vote to oppose...Sadly, DR-CAFTA is a step backward.
+
+* Fact: Like Senator Bernie Sanders, Hillary voted against the [Trade Act of 2002](http://www.politifact.com/truth-o-meter/statements/2016/apr/10/bernie-s/clinton-voted-virtually-every-trade-
+agreement-kill/). This legislation extended the Andean Trade Preference Act, but also gave the President fast track authority to negotiate trade deals. Fast track authority means that when the President negotiates a trade deal, Congress can only vote for or against it.
+
+* Fact: Hillary did not "flip flop" on TPP. Yes, Hillary did originally support the Trans-Pacific Partnership, or TPP. However, when the final deal was finished, [it did not meet her standards](http://www.ontheissues.org/2016/Hillary_Clinton_Free_Trade.htm). As a result, she felt obligated to oppose it:
+
+> I have always fought for the same values and principles, but, like most human beings--including those of us who run for office--I do absorb new information. I do look at what's happening in the world. Take the trade deal. I did say, when I was secretary of state, three years ago, that I hoped it would be the gold standard. It was just finally negotiated last week, and in looking at it, it didn't meet my standards. My standards for more new, good jobs for Americans, for raising wages for Americans. And I want to make sure that I can look into the eyes of any middle-class American and say, "this will help raise your wages." And I concluded I could not.
+
```

Comment:
Also, Hillary's statements in debates about "her standards" tend to be vague because they're intended for a mass audience. Digging through more of her rhetoric, it seems like her biggest standard is "enforceability". See http://www.ontheissues.org/2016/Hillary_Clinton_Free_Trade.htm for example. Having a trade prosecutor enforce breaches, having not only provisions that are meant to protect workers, but a way to actually follow-through, seems to be her standard.